src/Microsoft.AspNetCore.Authentication.Facebook/FacebookHandler.cs

```diff
@@ -78,6 +97,36 @@ public FacebookHandler(HttpClient httpClient)
                 identity.AddClaim(new Claim("urn:facebook:link", link, ClaimValueTypes.String, Options.ClaimsIssuer));
             }
 
+            var location = FacebookHelper.GetLocation(payload);
+            if (!string.IsNullOrEmpty(location))
+            {
+                identity.AddClaim(new Claim(ClaimTypes.StateOrProvince, location, ClaimValueTypes.String, Options.ClaimsIssuer));
```

Comment:
StateOrProvince? I don't think you have any guarantee about what's in this field.